portfolio/src/main/webapp/comments-page.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <title>Comments</title>
+      <script src = "comments.js" type="text/javascript"> </script>
+      <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Reenie+Beanie&display=swap" rel="stylesheet">
+      <link rel="stylesheet" href="style.css">
+  </head>
+  <body onload="getComment()">
+      <h2> Comments </h2>
+      <p> Leave me a comment here: </p>
+      <form action="/comments" method="POST">
+          <label for="name">Name:</label><br>
+          <input type="text" name="name-text"><br>
+          <label for="comment">Comment:</label><br>
+          <input type="text" name="comment-text"><br>
+          <input type="submit" value="Submit!" class="transparent-button">
```

Comment:
maybe make the inputs required so you can do "input type="text" name="name-text" required" that way people actually have to put both before submitting. Because right now i can just press submit in an empty field

Reply:
Good idea Many, will do.